tests/test_utils_modular.py

```diff
@@ -39,7 +39,7 @@ def candidate_test_is_on_temporary_notimplemented_list_cfe(context, expectation_
             # "expect_column_distinct_values_to_equal_set",
             # "expect_column_mean_to_be_between",
             # "expect_column_median_to_be_between",
-            "expect_column_quantile_values_to_be_between",
+            # "expect_column_quantile_values_to_be_between",
```

Comment:
@roblim I may not be understanding this: how come we are commenting out the quantile expectations, while modifying how the are configured (using "quantile_ranges" instead the max and min values used previously).  Thank you.

Reply:
The expectation is commented out here because it's an exclusion list - so if an expectation is uncommented, the tests are skipped.

The use of `quantile_ranges` actually isn't new - `min_value` and `max_value` were actually incorrect (prob. added reflexively since expectation has "_to_be_between" in name.